### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Motivation for this change

- lnch: 2017-02-16 -> unstable-2021-02-10
- mpd_clientlib: rename to libmpdclient
- tllist: 1.0.4 -> 1.0.5
- wbg: init at unstable-2020-08-01
- yambar: init at 1.6.1
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/wbg/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  pname = "wbg";
+  version = "unstable-2020-08-01";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/wbg";
+    rev = "1b05bd80d0f40e3ba1e977002d0653f532649269";
+    sha256 = "0i1j7aqvj0vl2ww5cvffqci1kjqjn0sw6sp2j0ljblaif6qk9asc";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson ninja scdoc git ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/wbg/default.nix:31]:
```suggestion
  nativeBuildInputs = [ pkg-config meson ninja scdoc git ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -17916,7 +17916,8 @@ in
   inherit (callPackages ../servers/mpd { })
     mpd mpd-small mpdWithFeatures;
 
-  mpd_clientlib = callPackage ../servers/mpd/clientlib.nix { };
+  mpd_clientlib = libmpdclient;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:17919]: Maybe we want to move this to alias.nix.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112662` at c2e7f9ad run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/77c6d553260a07e2781bad43225f1176">fcft</a></li></li>
    <li><a href="https://gist.github.com/5351c26bf8e7be1e27fdf6862895b6ca">foot</a></li></li>
    <li><a href="https://gist.github.com/32049d4c34a9dd105e594c9ad1b7d34e">fuzzel</a></li></li>
    <li>libmpdclient</li>
    <li><a href="https://gist.github.com/539cd19711e80453cbd7d418d8e903fe">lnch</a></li></li>
    <li><a href="https://gist.github.com/baa75c1f98f5dc5f977a73bde099fdc7">tllist</a></li></li>
    <li><a href="https://gist.github.com/3859ece2437afaae98604cc09e1fe0b3">wbg</a></li></li>
    <li><a href="https://gist.github.com/6ba38752550d8f930b856d2e8711c10d">yambar</a></li></li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112662` at c2e7f9ad run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/2454552b1cafc040a8d8f27238b7d5a5">fcft</a></li></li>
    <li><a href="https://gist.github.com/ce7c905cbd1319c6c892b9215a59c600">foot</a></li></li>
    <li><a href="https://gist.github.com/a6ff8709df62048a68d3bc91339f54f7">fuzzel</a></li></li>
    <li>libmpdclient</li>
    <li><a href="https://gist.github.com/f8032b8e606f8380c3df1d4bb2c4f9a7">lnch</a></li></li>
    <li><a href="https://gist.github.com/975cae7fdc8acbe30bba92d8a7ff83db">tllist</a></li></li>
    <li><a href="https://gist.github.com/9fc07e25bc19a0cae18c62e822bd1173">wbg</a></li></li>
    <li><a href="https://gist.github.com/d68cd679b4eff2f1fb2db3dbaf48b34d">yambar</a></li></li>
  </ul>
</details>

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (AndersonTorres) [reply pkgs/top-level/all-packages.nix:17919]: I will redo all these references in a future "treewide".

**Pull request merged**